monkey/infection_monkey/i_puppet/i_puppet.py

```diff
@@ -103,14 +103,16 @@ def fingerprint(
 
     @abc.abstractmethod
     def exploit_host(
-        self, name: str, host: str, options: Dict, interrupt: threading.Event
+        self, name: str, host: object, options: Dict, interrupt: threading.Event
     ) -> ExploiterResultData:
         """
         Runs an exploiter against a remote host
         :param str name: The name of the exploiter to run
-        :param str host: The domain name or IP address of a host
+        :param object host: The domain name or IP address of a host
```

Comment:
Just a note to remember to solve the circular dependency and change `object` to `VictimHost`.